intermediate_source/torch_export_tutorial.py

```diff
@@ -0,0 +1,533 @@
+#     ``torch.export`` and its related features are in prototype status and are subject to backwards compatibility
+#     breaking changes. This tutorial provides a snapshot of ``torch.export`` usage as of PyTorch 2.1.
+#
+# :func:`torch.export` is the PyTorch 2.X way to export PyTorch models into
+# standardized model representations, intended
+# to be run on different (i.e. Python-less) environments.
+#
+# In this tutorial, you will learn how to use :func:`torch.export` to extract
+# ``ExportedProgram``'s (i.e. single-graph representations) from PyTorch programs.
+# We also detail some considerations/modifications that you may need
+# to make in order to make your model compatible with ``torch.export``.
+#
+# **Contents**
+#
+# .. contents::
+#     :local:
+
+######################################################################
+# Basic Usage
+# -----------
+#
+# ``torch.export`` extracts single-graph representations from PyTorch programs
+# by tracing the target function, given example inputs.
+#
+# The signature of ``torch.export`` is:
+#
+# .. code:: python
+#
+#     export(
+#         f: Callable,
+#         args: Tuple[Any, ...],
+#         kwargs: Optional[Dict[str, Any]] = None,
+#         *,
+#         constraints: Optional[List[Constraint]] = None
+#     ) -> ExportedProgram
+#
+# ``torch.export`` traces the tensor computation graph from calling ``f(*args, **kwargs)``
+# and wraps it in an ``ExportedProgram``, which can be serialized or executed later with
+# different inputs. Note that while the output ``ExportedGraph`` is callable, it is not a
+# ``torch.nn.Module``. We will detail the ``constraints`` argument later in the tutorial.
+
+import torch
+from torch.export import export
+
+class MyModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.lin = torch.nn.Linear(100, 10)
+
+    def forward(self, x, y):
+        return torch.nn.functional.relu(self.lin(x + y), inplace=True)
+
+mod = MyModule()
+exported_mod = export(mod, (torch.randn(8, 100), torch.randn(8, 100)))
+print(type(exported_mod))
+print(exported_mod(torch.randn(8, 100), torch.randn(8, 100)))
+
+######################################################################
+# Let's review some attributes of ``ExportedProgram`` that are of interest.
+#
+# The ``graph`` attribute is an `FX graph <https://pytorch.org/docs/stable/fx.html#torch.fx.Graph>`__
+# traced from the function we exported, that is, the computation graph of all PyTorch operations.
+# The FX graph has some important properties:
+#
+# - The operations are "ATen-level" operations.
+# - The graph is "functionalized", meaning that no operations are mutations.
+#
+# The ``graph_module`` attribute is the ``GraphModule`` that wraps the ``graph`` attribute
+# so that it can be ran as a ``torch.nn.Module``.
+# We can use ``graph_module``'s ``print_readable``` to print a Python code representation
+# of ``graph``:
+
+print(exported_mod)
+exported_mod.graph_module.print_readable()
```

Comment:
Just figured out how to view the actual thing: https://docs-preview.pytorch.org/pytorch/tutorials/2557/intermediate/torch_export_tutorial.html

It looks like you need to do `print(exported_mod.graph_module.print_readable())`